grid/src/Overlays/exportdata/index.js

```diff
@@ -33,19 +33,24 @@ const ExportData = memo((props) => {
         isDownload = true;
         let filteredRow = [];
         let filteredRowValues = [];
+        let filteredRowHeader = [];
 
         setWarning("");
 
         if (managedColumns.length > 0 && downloadTypes.length > 0) {
-            rows.forEach((rowDetails) => {
+            let rowLength = rows.length;

```

Comment:
If value is not changing, use const

Reply:
Also add error handling:
const rowLength = rows &&  rows.length > 0 ? rows.length : 0;